client/components/backup-retention-management/index.tsx

```diff
@@ -149,14 +149,12 @@ const BackupRetentionManagement: FunctionComponent = () => {
 	}, [ currentRetentionPlan, retentionSelected ] );
 
 	useEffect( () => {
-		if ( updateRetentionRequestStatus === BACKUP_RETENTION_UPDATE_REQUEST.SUCCESS ) {
-			// Update the current retention plan to the one updated by the user.
-			setCurrentRetentionPlan( retentionSelected );
-			setConfirmationDialogVisible( false );
-		} else if ( updateRetentionRequestStatus === BACKUP_RETENTION_UPDATE_REQUEST.FAILED ) {
+		if (
+			updateRetentionRequestStatus === BACKUP_RETENTION_UPDATE_REQUEST.SUCCESS ||
+			updateRetentionRequestStatus === BACKUP_RETENTION_UPDATE_REQUEST.FAILED
+		) {
```

Comment:
Could be simplified with `includes()`, can't it (untested)? 
```suggestion
		if ( [ BACKUP_RETENTION_UPDATE_REQUEST.SUCCESS, BACKUP_RETENTION_UPDATE_REQUEST.FAILED ].includes( updateRetentionRequestStatus ) ) {
```

Reply:
I did try this, but it was throwing this error:
`Argument of type 'string' is not assignable to parameter of type '"success" | "failed"'.ts(2345)`

It seems related to the fact that we are returning a string from `getBackupRetentionUpdateRequestStatus()`, so we should probably add another type there to ensure the function is returning either one of the `BACKUP_RETENTION_UPDATE_REQUEST` values.